### ISSUE TITLE
monitoring multiple Asterisk servers events 

### ISSUE THREAD

**Issue by alaazaineldein** [AsterNET/AsterNET#246]
i got an issue in scalability in the below implementation
i have got multiple  Asterisk servers  and we create a hosted service to listen on the AMI handler events like the below
and this service is implemented using .net core 3.1 and deployed on AWS kubernetes pods 
currently i have got a dedicated pod for each  Asterisk server

events samples: 
manager.Join ,	manager.Leave ,manager.AgentCalled ,	manager.AgentComplete ,manager.Bridge ,manager.Dial and manager.OriginateResponse 

is there any recommendation to provide a scalable  solution ,your support is much appreciated

**Comment by Deantwo** [comment]
I am not sure what your question is. You just want advise on how best to manage multiple instances of AsterNET in parallel?
Do you have an issue with using multiple instances of AsterNET simultaneously?

**Comment by alaazaineldein** [comment]
Yes I need your advice about the best way to manage multiple instances of AsterNET in parallel

**Comment by Deantwo** [comment]
Best compared to what? Have you tried or tested it at all?
I can't see why there would be any issues with AsterNET running in parallel. So best I can suggest is that you just try to do it.
How you do it seem more based on how you make your application/service than anything else.
